rules/aip0133/http_uri_parent_test.go

```diff
@@ -15,59 +15,41 @@
 package aip0133
 
 import (
-	"strings"
 	"testing"
 
-	"github.com/golang/protobuf/proto"
-	dpb "github.com/golang/protobuf/protoc-gen-go/descriptor"
-	"github.com/jhump/protoreflect/desc/builder"
-	"google.golang.org/genproto/googleapis/api/annotations"
+	"github.com/googleapis/api-linter/rules/internal/testutils"
 )
 
 func TestHttpUriField(t *testing.T) {
 	tests := []struct {
 		testName   string
-		uri        string
-		methodName string
-		msg        string
+		URI        string
+		MethodName string
+		problems   testutils.Problems
 	}{
-		{"Valid", "/v1/{parent=publishers/*/books/*}", "CreateBook", ""},
-		{"InvalidVarParent", "/v1/{book=publishers/*/books/*}", "CreateBook", "`parent` field"},
-		{"NoVarParent", "/v1/publishers/*/books/*", "CreateBook", "`parent` field"},
-		{"Irrelevant", "/v1/{book=publishers/*/books/*}", "BuildBook", ""},
+		{"Valid", "/v1/{parent=publishers/*/books/*}", "CreateBook", testutils.Problems{}},
```

Comment:
Can you add a false positive case?

Reply:
The Valid case was the false positive case. The rule and test were both broken (because the regex was messed up).